lib/tracker/handlers/base.rb

```diff
@@ -13,6 +13,7 @@ class NotImplemented < StandardError; end
   def initialize(api_key:"", env:, uuid_fetcher: -> proc {}, uuid: nil)
     @api_key = api_key
     @env     = env
+    @request = Rack::Request.new(env)
```

Comment:
we can't use Rack::Request here since we want to be able to use this without tracker and this'll break in that case...i'm surprised that tests didn't break with this change

Reply:
@sent-hil here is the commit for the user `rack.input` https://github.com/WalkerAndCoBrandsInc/tracker/pull/17/commits/f5fc55513c9e11c877c8ad33289bade9b6296a51